sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -200,6 +200,20 @@ public partial class RequestContext
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
         public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct RequestFailedDetails
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public RequestFailedDetails(Azure.ResponseError? error, System.Collections.Generic.IDictionary<string, string>? additionalInfo) { throw null; }
+        public System.Collections.Generic.IDictionary<string, string>? AdditionalInfo { get { throw null; } }
+        public Azure.ResponseError? Error { get { throw null; } set { } }
+    }
+    public partial class RequestFailedDetailsParser
```

Comment:
Would `ErrorResponseParser` be a better name for this?  It would match the `Response.IsError` terminology elsewhere in the API.

Reply:
I don't feel strongly, but one reason I like the current name is that everything centers around `RequestFailedException`. Are there scenarios you are aware of where these concepts stand alone?

That said, if others like these names better, I'm good with it.

Reply:
I think it'll be good to decide in context once the rest of the API additions are closer to completion.